test/end-to-end/cases-coverage/expected/stylesheet/xsl-try-01-coverage.html

```diff
@@ -31,73 +31,126 @@ <h2>Contents</h2>
+069: <span class="whitespace">          </span><span class="missed">&lt;xsl:catch&gt;</span><span class="whitespace">                                                          </span><span class="comment">&lt;!-- Expected miss --&gt;</span>
+070: <span class="whitespace">            </span><span class="missed">&lt;xsl:text&gt;</span><span class="missed">999</span><span class="missed">&lt;/xsl:text&gt;</span><span class="whitespace">                                           </span><span class="comment">&lt;!-- Expected miss --&gt;</span>
+071: <span class="whitespace">          </span><span class="missed">&lt;/xsl:catch&gt;</span><span class="whitespace">                                                         </span><span class="comment">&lt;!-- Expected miss --&gt;</span>
+072: <span class="whitespace">        </span><span class="hit">&lt;/xsl:try&gt;</span>
+073: <span class="whitespace">      </span><span class="hit">&lt;/node&gt;</span>
+074: 
+075: <span class="whitespace">      </span><span class="comment">&lt;!-- Entire try/catch block not executed --&gt;</span>
+076: <span class="whitespace">      </span><span class="hit">&lt;xsl:if test="1 eq 2"&gt;</span>
+077: <span class="whitespace">        </span><span class="comment">&lt;!-- Using xsl:try select attribute - not executed --&gt;</span>
+078: <span class="whitespace">        </span><span class="missed">&lt;xsl:try select="999"&gt;</span><span class="whitespace">                                               </span><span class="comment">&lt;!-- Expected miss --&gt;</span>
+079: <span class="whitespace">          </span><span class="unknown">&lt;xsl:catch&gt;</span><span class="whitespace">                                                        </span><span class="comment">&lt;!-- Expected unknown --&gt;</span>
+080: <span class="whitespace">            </span><span class="unknown">&lt;xsl:map-entry key="999" select="()"/&gt;</span><span class="whitespace">                           </span><span class="comment">&lt;!-- Expected unknown --&gt;</span>
+081: <span class="whitespace">          </span><span class="unknown">&lt;/xsl:catch&gt;</span><span class="whitespace">                                                       </span><span class="comment">&lt;!-- Expected unknown --&gt;</span>
+082: <span class="whitespace">        </span><span class="missed">&lt;/xsl:try&gt;</span><span class="whitespace">                                                           </span><span class="comment">&lt;!-- Expected miss --&gt;</span>
+083: <span class="whitespace">        </span><span class="comment">&lt;!-- Using xsl:try sequence constructor - not executed, and xsl:catch has untraceable child --&gt;</span>
+084: <span class="whitespace">        </span><span class="unknown">&lt;xsl:try&gt;</span><span class="whitespace">                                                            </span><span class="comment">&lt;!-- Expected unknown --&gt;</span>
```

Comment:
This line (84 in the XSLT) and line 143 (91 in the XSLT) are related to [my comment ](https://github.com/xspec/xspec/issues/2126#issuecomment-2994432950). They're marked 'unknown' here but 'missed' would be better.